src/schema/v2/viewingRoom.ts

```diff
@@ -66,6 +65,8 @@ defineCustomLocale(LocaleEnViewingRoomRelativeLong, {
 export const ViewingRoomType = new GraphQLObjectType<any, ResolverContext>({
   name: "ViewingRoom",
   fields: () => {
+    const { PartnerArtworks } = require("schema/v2/partner/partnerArtworks")
+
```

Comment:
**question (non-blocking):** Trying to get my head around the `require` (vs `import`) — this is because it's being referenced inside of the thunk declared on L67?